boot/zephyr/CMakeLists.txt

```
@@ -352,3 +352,14 @@ zephyr_library_sources(
   ${BOOT_DIR}/zephyr/nrf_cleanup.c
 )
 endif()
+
+if(SYSBUILD AND CONFIG_PCD_APP)
+    # Sysbuild requires details of the RAM flash device are stored to the cache of MCUboot so
+    # that they can be read when running partition manager
```

nit: Tab size and wording
```suggestion
  # Sysbuild requires that details of the RAM flash device are stored to the cache of MCUboot so
  # that they can be read when running partition manager
```